projects-to-be-tested/karaf/camel-log/src/main/java/org/eclipse/jkube/quickstart/karaf/OrderGenerator.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.quickstart.karaf;
```

Comment:
I think package for your integration tests should be `org.eclipse.jkube.integrationtests.*` just to be consistent with other test-projects